### DISCUSSION TITLE
[ARCHIVED] Exercise 1 - Virtual Exhibition

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#1] [Homework]
- create at least 3 different `Plane` game objects
- scale them, rotate them and position them to create floors and walls
- import at least 5 images and position them across the wall, as if they were hanging
- (bonus) import 1 sound file and add it to the scene

Post your screenshots here!

**Comment by miafan716** [comment]
Screenshot:
<img width="960" alt="AtG Assignment W2-Mia" src="https://user-images.githubusercontent.com/55967980/151711455-971ba9c1-d52f-4e44-b4a1-b730899a1c21.png">

Video demo for bonus:

https://user-images.githubusercontent.com/55967980/151711474-b22ea607-ece8-41a9-852d-a745b82c1935.mp4

[Tutorial I followed for Audio Triggering](https://www.youtube.com/watch?v=DMJfQbgz_zo)

**Comment by aidanmassie** [comment]
Screenshot:

<img width="1440" alt="Screen Shot 2022-02-01 at 12 07 05 PM" src="https://user-images.githubusercontent.com/60477421/152015894-2eddf13b-0310-4998-86dd-8e6965347957.png">

**Comment by yylinj** [comment]
screenshot
![image](https://user-images.githubusercontent.com/98582179/152044668-50bb5782-63f9-47b6-bab3-1fd6629a21e1.png)

**Comment by mdl601** [comment]
Michelle Leidecker Screenshot and video 

<img width="1200" alt="Michelle Leidecker_ Screenshot Ex 1" src="https://user-images.githubusercontent.com/98850518/152051542-9c97e427-61fb-4ed0-b4b5-716077562aba.png">

https://user-images.githubusercontent.com/98850518/152051559-0fab928c-4fb0-45a6-82e4-3e433e66322d.mov

**Comment by joshblecherman** [comment]
<img width="1440" alt="Screen Shot 2022-02-01 at 11 06 12 PM" src="https://user-images.githubusercontent.com/57162769/152060214-017af651-cd13-40ce-ba99-2a69f9a61072.png">

**Comment by Lkochar19** [comment]
<img width="930" alt="Screen Shot 2022-02-01 at 11 19 09 PM" src="https://user-images.githubusercontent.com/89488715/152061203-85d59d3f-4b24-4520-bd24-1c7fd177ca14.png">

**Comment by vb1424** [comment]
<img width="1434" alt="Screen Shot 2022-02-02 at 4 19 18 AM" src="https://user-images.githubusercontent.com/98861299/152083236-294aacfd-688b-469f-9996-0741a8f5d512.png">

**Comment by zanderzel** [comment]
<img width="1440" alt="Screen Shot 2022-02-01 at 9 52 33 PM" src="https://user-images.githubusercontent.com/58765528/152086136-aa6772cf-4e86-4a05-b1e3-085639249b9a.png">

**Comment by palomabrites** [comment]
<img width="1440" alt="img" src="https://user-images.githubusercontent.com/98867529/152097881-84d46a88-23b0-4bba-b51f-1d548078f23f.png">

**Comment by JaviC122** [comment]
<img width="1184" alt="Screen Shot 2022-02-02 at 9 34 53 AM" src="https://user-images.githubusercontent.com/98875799/152120270-c230b8ec-0975-4416-910e-c63b4097323c.png">

**Comment by fractalessense** [comment]
<img width="1440" alt="Screen Shot 2022-02-02 at 23 00 02" src="https://user-images.githubusercontent.com/98880710/152134399-3e02f72f-37b3-4a5a-8c70-938a31be7ef2.png">

**Comment by chloeumlaut** [comment]
<img width="1157" alt="Screen Shot 2022-02-02 at 6 52 32 PM" src="https://user-images.githubusercontent.com/98907766/152209773-ef3ae38d-1063-4657-ae08-63d9cbe3367e.png">

**Comment by mariaabzm** [comment]
<img width="571" alt="Screen Shot 2022-09-13 at 1 56 35 PM" src="https://user-images.githubusercontent.com/113439496/189894767-f431cd0d-beab-4c7b-bfc6-42d5a717f324.png">
<img width="866" alt="Screen Shot 2022-09-13 at 1 57 40 PM" src="https://user-images.githubusercontent.com/113439496/189894947-b33fb6bb-67b8-4107-a2f2-cb1896080717.png">

**Comment by olivestew** [comment]
cat lounge
<img width="658" alt="Screenshot 2022-09-13 at 20 12 19" src="https://user-images.githubusercontent.com/91790610/189978860-34638531-a2d7-4e61-ad9c-b0cd19c0deff.png">

**Comment by izzyrieken** [comment]
<img width="700" alt="Screen Shot 2022-09-13 at 8 16 37 PM" src="https://user-images.githubusercontent.com/58444126/189979557-5eff30ab-112e-4a24-bdd2-37896aff8feb.png">

Klimt Gallery

**Comment by Alupie** [comment]
![Screen Shot 2022-09-13 at 8 33 28 PM](https://user-images.githubusercontent.com/113468588/189982785-0b53e6d8-5107-4a9e-bf66-a8d50e913f0e.png)

**Comment by Dav1dyang** [comment]
<img width="1292" alt="Screen Shot 2022-09-13 at 8 34 41 PM" src="https://user-images.githubusercontent.com/55843005/189983430-472956ad-8e56-484d-91f9-c6589dc1791f.png">

The audio plays when the space bar is pressed. The audio stopped when the key "S" is pressed. The artworks are by Liana Finck.

**Comment by kylewenchang** [comment]
<img width="989" alt="Screen Shot 2022-09-13 at 10 21 21 PM" src="https://user-images.githubusercontent.com/63214975/190002227-471653ff-3d32-42c4-9562-4eadd9242fa0.png">

Video: https://www.youtube.com/watch?v=n8gyM6qLExU

I added Quixel textures to the walls and floor and added a cylinder with a light for an overhead light. The images were made with an online image generator, the music is a recording of a vinyl I had on my laptop.

**Comment by IanPort** [comment]
<img width="593" alt="Museum1" src="https://user-images.githubusercontent.com/113472376/190031992-579041fc-c96e-4c6e-a01f-f626b5ab8ea7.png">
<img width="1209" alt="MuseumGame" src="https://user-images.githubusercontent.com/113472376/190031998-efb93865-375c-4fad-b28c-fce721479dd1.png">
<img width="1205" alt="MuseumGame2" src="https://user-images.githubusercontent.com/113472376/190032000-be62f81a-2b64-49a1-a63a-676b0715dd79.png">

A collection of video, projection, neon and interactive art I have been reflecting on recently (including one personal project). 
Audio: Luv, by Giraffage

**Comment by kenparks12** [comment]
<img width="1792" alt="Screen Shot 2022-09-14 at 02 52 48" src="https://user-images.githubusercontent.com/97991944/190034862-01392893-e782-4b3a-abd3-ee35c2346a97.png">
collection of my favorite things!

**Comment by vasantsun29** [comment]
<img width="877" alt="Screen Shot 2022-09-14 at 8 38 49 AM" src="https://user-images.githubusercontent.com/113478467/190078967-3911317c-9abb-46b1-818a-a04fe61ad22c.png">

**Comment by luyumenq** [comment]
<img width="1728" alt="Screenshot 2022-09-14 at 11 29 31" src="https://user-images.githubusercontent.com/72938590/190117350-a321ccc0-93dc-44ea-8278-d65ce30d5ea6.png">
<img width="1728" alt="Screenshot 2022-09-14 at 11 30 48" src="https://user-images.githubusercontent.com/72938590/190117705-06f254b5-774d-4ef2-85f1-04e2878bb2e4.png">

**Comment by dovebardin** [comment]
<img width="738" alt="Screen Shot 2022-09-14 at 11 35 16 AM" src="https://user-images.githubusercontent.com/113513286/190119520-58c049b9-68f4-402c-a2f2-848508f773cd.png">

**Comment by mp5822** [comment]
<img width="1440" alt="Screen Shot 2022-09-18 at 1 50 15 PM" src="https://user-images.githubusercontent.com/113290934/190900761-e7f469a2-32d9-4145-8893-a02c5347fb17.png">

**Comment by Yupu-Chen** [comment]
<img width="1162" alt="exercise 1" src="https://user-images.githubusercontent.com/112383893/215262305-dfa93e39-1f67-4d79-8bb9-8cfb422e6a4a.png">

**Comment by Opal42** [comment]
![image_003_0000](https://user-images.githubusercontent.com/123597298/215796888-415989b7-efce-460a-ae85-36d5f46b6714.jpg)

https://user-images.githubusercontent.com/123597298/215797167-9e68dcc5-b738-4a2b-a64f-0b7f1f241580.mp4

**WELCOME TO GOOSE GALLERY**
Textures for the floor and walls were taken from the Unity Asset Store, and tiled to (hopefully) reasonable proportions. The audio source playing the lovely Goosecall is tied to the centre painting, with spatial rolloff. I stuck with the default rolloff curve, so the effect is not as obvious in this clip, but would like to experiment more with it in the future. Coming from a Rhino background, I've found it frustrating not to be able to snap to object, and can't find any in-built ways to do it (only snap to grid).

**Comment by mariamabdelrazekk** [comment]
<img width="1440" alt="spotify gallery 1" src="https://user-images.githubusercontent.com/91086698/215910522-a987e398-74cd-4f6c-bab3-76112c7f9f8b.png">

Welcome to my Spotify Wrapped!
Since music is a huge part of my life, I have decided to base my first project in Unity on my Spotify wrapped. The "Gallery" represents albums that I have listened to the most in 2022. I did not curate this gallery very well as I found it kind of hard to navigate how to place the walls accurately next to each other to create a room. However, I placed the sphere in the middle of the room in order to give the room a "center".

**Comment by aacarcrash** [comment]
<img width="1429" alt="Screenshot 2023-02-01 at 3 06 20 AM" src="https://user-images.githubusercontent.com/29751759/215927433-fbb5c8be-6be5-4ebf-8673-247b681fc3a2.png">
<img width="1418" alt="Screenshot 2023-02-01 at 3 06 00 AM" src="https://user-images.githubusercontent.com/29751759/215927441-511c651c-6bd0-411f-99bb-b2eaf662e6eb.png">

This is an abstract art gallery. The sound emanates from the central abstract sculpture. 
[Link to Video Demo](https://youtu.be/jTMW48ifHKc)

**Comment by hannah-y-zhao** [comment]
<img width="1440" alt="Screen Shot 2023-02-01 at 2 57 40 AM" src="https://user-images.githubusercontent.com/92940850/215928465-1cd32fc4-fde1-46fe-adbd-1a4d77e807a4.png">
This is a gallery of my cat.
Unfortunately I couldn't get my audio to work, and I'm not sure if it was something with my import settings or just the audio file itself.

**Comment by daniaezz** [comment]
<img width="1440" alt="Screen Shot 2023-02-01 at 8 32 48 AM" src="https://user-images.githubusercontent.com/90758857/215979602-562b9391-829a-4e4c-bf40-dfeecdd55ac9.png">
<img width="1440" alt="Screen Shot 2023-02-01 at 8 38 32 AM" src="https://user-images.githubusercontent.com/90758857/215980390-49623000-0604-4fbf-95f9-76f6c45137ca.png">

**Comment by cz2109** [comment]
![CatherineZhu_UnityAssignment1_Screenshot](https://user-images.githubusercontent.com/123963318/215983294-bd062d95-1fe4-4e88-83d5-9482005dfd48.png)

This is a gallery of betta fish. I love using betta fish as an artistic motif and my Pinterest board was full of them so it was easy to gather the images. When played, there is an audio clip of ambient bubbles :) My only concern is that I did not remember how to snap pieces together, so there are minuscule gaps throughout and I would love to sort that out.

**Comment by emersonmack** [comment]
[Screen Shot 2023-02-01 at 3.45.35 PM.zip](https://github.com/periode/augmenting-gallery/files/10560119/Screen.Shot.2023-02-01.at.3.45.35.PM.zip)
![Screen Shot 2023-02-01 at 3 46 19 PM](https://user-images.githubusercontent.com/123585567/216128068-fbd8ff33-435c-49b2-a8b3-73dee44b1288.png)

This Gallery is a collection of Keith Haring's works. I find his art simply colorful and childlike. I organized the collection to be a gallery within a larger museum with two ways in and out of the space.
